src/App.tsx

```diff
@@ -1,10 +1,69 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { FilteredBy } from './types/FilteredBy';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { TodoMain } from './components/TodoMain/TodoMain';
+import { TodoError } from './components/TodoError/TodoError';
+import { deleteTodos, getTodos } from './api/todos';
+import { TodoErrorType } from './types/TodoErrorType';
 
-const USER_ID = 0;
+const USER_ID = 11136;
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>([]);
+  const [filteredBy, setFilteredBy] = useState(FilteredBy.All);
+  const [hasError, setHasError] = useState(TodoErrorType.noError);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleDeleteTodo = (todoId: number) => {
+    setIsLoading(true);
+    setHasError(TodoErrorType.noError);
+
+    deleteTodos(todoId)
+      .then(() => {
+        setIsLoading(false);
+        setHasError(TodoErrorType.noError);
+        setTodosFromServer(currentTodos => currentTodos.filter(
+          todo => todo.id !== todoId,
+        ));
+      })
+
+      .catch(() => {
+        setIsLoading(false);
+        setHasError(TodoErrorType.deleteTodoError);
+      });
+  };
+
+  useEffect(() => {
+    setHasError(TodoErrorType.noError);
+
+    getTodos(USER_ID)
+      .then(setTodosFromServer)
+      .catch(() => {
+        setHasError(TodoErrorType.loadTodoError);
+        setTimeout(() => {
+          setHasError(TodoErrorType.noError);
+        }, 3000);
```

Comment:
move to `TodoError` on mount

Reply:
[comment](https://github.com/mate-academy/react_todo-app-loading-todos/pull/680/files#r1278261541) not fixed